contracts/v2/paranets/ParanetNeuroIncentivesPool.sol

```diff
@@ -281,8 +282,16 @@ contract ParanetNeuroIncentivesPool is Named, Versioned {
         );
     }
 
-    function claimKnowledgeMinerReward() external onlyParanetKnowledgeMiner {
-        ParanetKnowledgeMinersRegistry pkmr = paranetKnowledgeMinersRegistry;
+    function getTotalKnowledgeMinerIncentiveEstimation() public view returns (uint256) {
+        uint96 unrewardedTracSpent = paranetKnowledgeMinersRegistry.getUnrewardedTracSpent(msg.sender, parentParanetId);
+
+        if (unrewardedTracSpent < TOKENS_DIGITS_DIFF) {
+            revert ParanetErrors.MinimalRewardedTracAmountNotMet(
```

Comment:
Should we revert here or return 0? @branarakic

Reply:
since its a getter I would rather return 0 than escalate to an error

Reply:
This getter is used in the claim function, which then wouldn't revert but there will be no reward awarded. Might be confusing

Reply:
are you sure @u-hubar? you mean claimKnowledgeMinerReward()? 
cause from what I see that one reverts if claimableNeuroReward == 0 (L343)

Reply:
(I like L343 implementation btw - claiming reverting but getter returning 0)

Reply:
@branarakic, it will revert, but with a different reason (because of 0), might be confusing that there is some spent TRAC, but reward is 0

Reply:
Anyway, I'm not strongly against, so let's just decide what would be the best from the user's perspective

Reply:
Could be ok to have 0 returned, but just making clear on the UI level later that minimum rewarded amount of TRAC spent is 10^6